partcad/src/partcad/sketch_factory_alias.py

```
@@ -62,21 +63,26 @@ def __init__(self, ctx, source_project, target_project, config):
 
             pc_logging.debug("Initializing an alias to %s" % self.source)
 
-    async def instantiate(self, sketch):
-        with pc_logging.Action("Alias", sketch.project_name, f"{sketch.name}:{self.source_sketch_name}"):
+    async def instantiate(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `instantiate`

(ANN201)